src/services/user.ts

```diff
@@ -0,0 +1,7 @@
+import usersFromServer from '../api/users';
+import { User } from '../types/User';
+
+export function getUserById(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
```

Comment:
It's not necessary to wrap it here 
```suggestion
  return usersFromServer.find(user => user.id === userId) || null;
```